src/Sylius/Bundle/CoreBundle/spec/Form/DataTransformer/ProductTaxonToTaxonTransformerSpec.php

```diff
@@ -0,0 +1,76 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace spec\Sylius\Bundle\CoreBundle\Form\DataTransformer;
+
+use PhpSpec\ObjectBehavior;
+use Sylius\Bundle\CoreBundle\Form\DataTransformer\ProductTaxonToTaxonTransformer;
+use Sylius\Component\Core\Model\ProductTaxonInterface;
+use Sylius\Component\Core\Model\TaxonInterface;
+use Sylius\Component\Resource\Factory\FactoryInterface;
+use Symfony\Component\Form\DataTransformerInterface;
+
+/**
+ * @author Anna Walasek <anna.walasek@lakion.com>
+ */
+final class ProductTaxonToTaxonTransformerSpec extends ObjectBehavior
+{
+    function let(FactoryInterface $productTaxonFactory)
+    {
+        $this->beConstructedWith($productTaxonFactory);
+    }
+
+    function it_is_initializable()
+    {
+        $this->shouldHaveType(ProductTaxonToTaxonTransformer::class);
+    }
+    
+    function it_implements_data_transformer_interface()
+    {
+        $this->shouldImplement(DataTransformerInterface::class);
+    }
+
+    function it_transforms_product_taxon_to_taxon(
+        ProductTaxonInterface $productTaxon,
+        TaxonInterface $taxon
+    ) {
+        $productTaxon->getTaxon()->willReturn($taxon);
+
+        $this->transform($productTaxon)->shouldReturn($taxon);
+    }
+
+    function it_returns_null_during_transform()
```

Comment:
``it_returns_null_during_transforming_null_value``? As it seems like it does it every time 😄